integration/specs/InternalOverlay/internalOverlay-1.spec.js

```diff
@@ -31,4 +46,53 @@ describe('InternalOverlay with default position resolver', () => {
         $('[data-preview="InternalOverlay"]').click();
         expect(internalOverlay.exists()).toBe(false);
     });
+
+    it('should set focus on trigger when press tab key', () => {
+        const button = new PageInternalOverlayButton(BUTTON);
+        $('[data-preview="InternalOverlay"]').click();
+        browser.keys(TAB_KEY);
+        expect(button.hasFocusTrigger()).toBe(true);
+    });
+
+    it('should exist after set focus on trigger and press enter key', () => {
+        const button = new PageInternalOverlayButton(BUTTON);
+        const internalOverlay = new PageInternalOverlay(OVERLAY);
+        $('[data-preview="InternalOverlay"]').click();
```

Comment:
where is this element?

Reply:
@rgah2107 I think the best solution is to add an id to the <Container> of the example, this way you can use it as a trigger for the OutsideClick.